tests/unit/EligibleAssetRemoved.test.ts

```diff
@@ -0,0 +1,101 @@
+import {
```

Comment:
I suggest maybe to
1. Merge prettier config in develop
2. Format the test and utils from your previous PR and merge the new formating
3. Tackle this PR

Reply:
yeah i can that indeed